config/app.php

```diff
@@ -43,9 +43,11 @@
         Blumilk\Meetup\Core\Providers\EventServiceProvider::class,
         Blumilk\Meetup\Core\Providers\RouteServiceProvider::class,
         Blumilk\Meetup\Core\Providers\TelescopeServiceProvider::class,
+        Laravel\Socialite\SocialiteServiceProvider::class,
     ],
     "aliases" => [
         Facade::defaultAliases()->toArray(),
         "Formats" => Blumilk\Meetup\Core\Formats::class,
+        "Auth" => Illuminate\Support\Facades\Auth::class,
```

Comment:
You are duplicating `Auth` facade here. The entire `aliases` are wrongly build, because you're adding result of `defaultAliases()->toArray()` to array. You should merge app's aliases with Laravel ones.

`"aliases" => [...Facade::defaultAliases()->toArray(), "Formats" => Blumilk\Meetup\Core\Formats::class]```

Something like this should be fine. Of course after you'd format it properly.

Reply:
It's solved in #38.

Reply:
Fixed in #38